ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -47,17 +274,22 @@ class GeneticAlgorithm(RavenSampled):
   """
     This class performs Genetic Algorithm optimization ...
   """
-  convergenceOptions = {'objective': r""" provides the desired value for the convergence criterion of the objective function
+  convergenceOptions = {'objective': r""" provides the desired value or values for the convergence criterion of the objective function
                         ($\epsilon^{obj}$). In essence this is solving the inverse problem of finding the design variable
-                         at a given objective value, i.e., convergence is reached when: $$ Objective = \epsilon^{obj}$$.
-                        \default{1e-6}, if no criteria specified""",
+                         at a given objective value, i.e., convergence is reached when: $$ Objective = \epsilon^{obj}$$
+                         For multiobjective problems, a comma separated list of objective
+                         values should be provided instead of a single value.
+                        \default{1e-6}, if no criteria specified.""",
                         'AHDp': r""" provides the desired value for the Average Hausdorff Distance between populations""",
                         'AHD': r""" provides the desired value for the Hausdorff Distance between populations""",
                         'HDSM': r""" provides the desired value for the Hausdorff Distance Similarity Measure between populations.
                                      This convergence criterion is based on a normalized
                                      similarity metric that can be summurized as the normalized Hausdorff distance
                                      (with respect the domain of to population/iterations). The metric is normalized between 0 and 1,
                                      which implies that values closer to 1.0 represents a tighter convergence criterion."""}
+  ##TODO: Explore MOEA/D (Multi-Objective Evolutionary Algorithm based on Decomposition) or
+  # PESA-II (Pareto Envelope-Based Selection Algorithm II)
+  # These algorithms can offer better performance and robustness in certain scenarios
```

Comment:
Please add references for these methods.

Reply:
@Jimmy-INL  Can you fix this or send the reference to me?  Thanks.